@AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mfdConstants.h

```diff
@@ -181,13 +181,14 @@
 #define MFD_IND_TSD_SHOW_WPT_DATA_CURRTE 0
 #define MFD_IND_TSD_SHOW_ENEMY_UNITS 1
 #define MFD_IND_TSD_SHOW_FRIENDLY_UNITS 2
-#define MFD_IND_TSD_SHOW_PLAN_TGTS 3
-#define MFD_IND_TSD_SHOW_CTRLM 4
-#define MFD_IND_TSD_SHOW_SHOT 5
-#define MFD_IND_TSD_SHOW_HAZ 6
-#define MFD_IND_TSD_SHOW_WIND 7
-#define MFD_IND_TSD_SHOW_ENDR 8
-#define MFD_IND_TSD_ASE_FOOTPRINT 9
+#define MFD_IND_TSD_SHOW_RLWR 3
+#define MFD_IND_TSD_SHOW_PLAN_TGTS 4
+#define MFD_IND_TSD_SHOW_CTRLM 5
+#define MFD_IND_TSD_SHOW_SHOT 6
+#define MFD_IND_TSD_SHOW_HAZ 7
+#define MFD_IND_TSD_SHOW_WIND 8
+#define MFD_IND_TSD_SHOW_ENDR 9
+#define MFD_IND_TSD_ASE_FOOTPRINT 10
```

Comment:
Why move these all along rather than add all to the end?

Reply:
It was a git revert